pkg/v1/webhooks/certificates.go

```diff
@@ -63,53 +63,70 @@ func addCertsToWebhookConfigs(ctx context.Context, k8sclient kubernetes.Interfac
 	return nil
 }
 
-func NewTLSSecret(ctx context.Context, secretName, serviceName, certPath, keyPath, namespace string) (*corev1.Secret, error) {
-	secret, err := resources.MakeSecretInternal(ctx, secretName, namespace, serviceName)
-	if err != nil {
-		return nil, err
-	}
+func WriteServerTLSToFileSystem(ctx context.Context, certPath, keyPath string, secret *corev1.Secret) error {
+	// Writes servers certificate and key in provided secret to the filesystem paths provided
```

Comment:
Godoc needs to be before the function signature to be rendered properly in https://pkg.go.dev/github.com/vmware-tanzu/tanzu-framework. Should be of the form:

```go
// FunctionFoo does something.
func FunctionFoo() {}
```

Example: https://github.com/vmware-tanzu/tanzu-framework/blob/db9d5fb7dfbbbca8667bcc9ab0d16b0874d7057f/pkg/v1/sdk/features/featuregate/featuregate.go#L96-L97

Reply:
done